test/e2e/tests/notifications/constants.ts

```diff
@@ -5,3 +5,6 @@ export const NOTIFICATIONS_TEAM_PASSWORD = 'notify_password';
 // You can use the storage key below to generate mock data
 export const NOTIFICATIONS_TEAM_STORAGE_KEY =
   '0d55d30da233959674d14076737198c05ae3fb8631a17e20d3c28c60dddd82f7';
+
+export const NOTIFICATIONS_TEAM_IMPORTED_PRIVATE_KEY =
+  '0179f7453ff337aba89d04b00085764092cf8c0cfe91d5614c39c2be902ad582';
```

Comment:
shouldn't we avoid to hardcorde private keys in code? instead, using them as parameters (for example as a GHA secret in CI)

Reply:
As these are used for testing and no risk in losing funds there is no risk to have them there. Having to depend on managing secrets could end up being hard to manage in this repo i believe. This is also the same pattern used in other tests.

Reply:
LGTM. Approved

Reply:
Haha, yes this account is only for testing purposes and has no funds on it.

To keep this PR unblocked, can we add some doc comments/inline comments to explain what this private key is used for. I can imagine others who find this will start firing red flags 😆